config/config_test.go

```diff
@@ -17,8 +17,8 @@ func TestSpec(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 			Convey("The values should be set to the expected defaults", func() {
-				So(cfg.AwsRegion, ShouldEqual, "eu-west-1")
-				So(cfg.AwsService, ShouldEqual, "es")
+				So(cfg.Aws.Region, ShouldEqual, "eu-west-1")
```

Comment:
Adding a test for `Aws.Filename`, `Aws.Profile` and `Aws.TLSInsecureSkipVerify`
But most importantly `Aws.TLSInsecureSkipVerify` as it should never be true but only for development. That little extra